bot/cogs/tags.py

```diff
@@ -146,6 +147,7 @@ def _command_on_cooldown(tag_name: str) -> bool:
                     description="**There are no tags in the database!**",
                     colour=Colour.red()
                 ))
+                return True
```

Comment:
It'd be better to return `False` so that it can try to suggest a command instead. The embed saying "there are no tags" would still be visible, but I think it should just not be shown at all if it's being called from the error handler (can add a boolean parameter to the function to distinguish it).

Reply:
That's correct, I think i meant to do this, my bad

Reply:
This still needs to be addressed.

Reply:
I changed the implementation to return false, I will check what is wrong.

Reply:
I think you are confused about what this comment refers to. It's specifically regarding an embed being sent with the message `"**There are no tags in the database!**"`, which I'm suggesting should _not_ be sent at all if called from the error handler (i.e. merely returning `False` will not be adequate).

Reply:
Oh ok, but we also need to return false at the end of the function, correct?

Reply:
Don't think it matters since that line will never be reached (every branch of execution already has a return statement), but I suppose no harm in keeping it.

Reply:
Yeah, correct

Reply:
Like I said, 

> merely returning False will not be adequate

The **embed should not be sent** when this function is called by the error handler. I've proposed the solution of adding a parameter to indicate that.

Reply:
Both **Current tags** and **There are no tags in the database!** embeds shouldn't be sent if the error handler invoked the function, correct?

Reply:
Sorry, never mind. I just realised those only get sent if `tag_name` is None. The error handler never sends `None` as the name so nothing needs to be changed.

Reply:
Yeah, that makes sense, cool, releasing latest commits and i think we are there